config/environments/production.rb

```diff
@@ -2,23 +2,23 @@
 Rails.application.configure do
   config.cache_classes = true
   config.eager_load = true
-  config.consider_all_requests_local       = false
+  config.consider_all_requests_local = false
   config.action_controller.perform_caching = true
-  config.serve_static_files = ENV['RAILS_SERVE_STATIC_FILES'].present?
+  config.serve_static_files = Figaro.env.serve_static_files == true
```

Comment:
Figaro values are always String objects afaik, so the `true` may need to be `'true'`.